v0/destinations/rockerbox/data/RockerboxTrackConfig.json

```diff
@@ -5,24 +5,30 @@
     "sourceFromGenericMap": true
   },
   {
-    "sourceKeys": ["context.traits.email", "properties.email"],
+    "sourceKeys": ["email"],
```

Comment:
No array required if its a single key (sourceKey)

Reply:
fixed the same